javatests/com/williamfiset/datastructures/splaytree/SplayTreeTest.java

```diff
@@ -0,0 +1,55 @@
+package javatests.com.williamfiset.datastructures.splaytree;
```

Comment:
You can remove this file + directory now.

Reply:
Done